app/views/wiki_menu_items/edit.html.erb

```diff
@@ -57,7 +57,7 @@ See doc/COPYRIGHT.rdoc for more details.
       <%= form.check_box 'new_wiki_page' %>
       <%= form.label 'new_wiki_page', l(:label_wiki_show_new_page_link) %>
 
-      </br>
+      <br>
```

Comment:
Earlier in my life, when I stopped looking at HTML, the new thing was that self-closing tags should be apparent. Done like this: `<br />` (I think what the original author intended to do).

At the time, `<br>` was the old thing, that was deprecated.

What is the current way of doing things? I thought `<br />` was still hot...